wordpress-mobile/MediaPicker-iOS
# Full Screen Preview

Fixes #39 

This PR implements a full screen preview for assets in the libraries.

How to test: 
 - To active the preview just to a long press on an asset on the collection.
 - Test with different orientation on different devices.
 - Test with iCloud assets to see if loading happens correctly
 - Test deleting asset while doing preview.

The main idea on this PR was to implement the simplest implementation for preview, so no extras like:

- control of video
- editing of asset
- switching between assets in full screen

These things could be implemented in other future PR's.
